templates/security/cves/about.html

```diff
@@ -0,0 +1,438 @@
+            <div class="p-side-navigation__drawer-header">
+              <a href="#"
+                 class="p-side-navigation__toggle--in-drawer js-drawer-toggle"
+                 aria-controls="drawer">Toggle table of contents</a>
+            </div>
+            <ul class="p-side-navigation__list">
+              <li class="p-side-navigation__item">
+                <a class="p-side-navigation__link highlight-link is-active" href="#cves">What are CVEs?</a>
+              </li>
+              <li class="p-side-navigation__item">
+                <a class="p-side-navigation__link highlight-link" href="#security">Getting security updates</a>
+              </li>
+              <li class="p-side-navigation__item">
+                <a class="p-side-navigation__link highlight-link" href="#repos">Repositories</a>
+              </li>
+              <li class="p-side-navigation__item">
+                <a class="p-side-navigation__link highlight-link" href="#priority">Priority levels</a>
+              </li>
+              <li class="p-side-navigation__item">
+                <a class="p-side-navigation__link highlight-link" href="#statuses">Statuses</a>
+              </li>
+              <li class="p-side-navigation__item">
+                <a class="p-side-navigation__link highlight-link" href="#export-data">Exporting CVE data</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      <div class="col-9">
+        <h2 class="question-heading" id="cves">What are CVEs?</h2>
+        <div class="p-section">
+          <p>
+            The Common Vulnerabilities and Exposures (CVE) system is used to identify, define, and catalogue publicly disclosed cybersecurity vulnerabilities.
+            Canonical keeps track of all CVEs affecting Ubuntu, and releases a <a href="/security/notice">security notice</a> when an issue is fixed.
+          </p>
+          <p>
+            In our <a href="/security/cves">CVE database</a>, you can check which packages are affected by a given vulnerability for every supported Ubuntu version,
+            and their statuses. We may also provide notes that can help you to better understand the impact of these vulnerabilities in an Ubuntu environment,
+            and how we have fixed the issue.
+          </p>
+        </div>
+
+        <h2 class="question-heading" id="security">Getting security updates</h2>
+        <div class="p-section">
+          <p>
+            Ubuntu Long Term Support (LTS) releases get up to 12 years of security maintenance and support. The initial <a href="/about/release-cycle">5 years</a>
+            of standard security updates for packages in the <a href="https://help.ubuntu.com/community/Repositories/Ubuntu">Ubuntu Main</a> repository comes out
+            of the box together with fixes for packages in the Ubuntu Universe repository coming from the Ubuntu community and Debian.
+          </p>
```

Comment:
There's a paragraph missing here "Additional security coverage from Canonical for all packages..."